scripts/release/rpm/ubi.dockerfile

```diff
@@ -18,7 +18,7 @@ COPY . .
 # We have to explicitly specify 'python39' to install Python 3.9.
 RUN --mount=type=secret,id=PIP_INDEX_URL export PIP_INDEX_URL=$(cat /run/secrets/PIP_INDEX_URL) && \
     dos2unix ./scripts/release/rpm/azure-cli.spec && \
-    REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=$python_package PYTHON_CMD=python3.9 \
+    REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=$python_package PYTHON_CMD=$python_package \
```

Comment:
In python3.9, the `PYTHON_PACKAGE` is `python39`, `PYTHON_CMD` is `python3.9`.

In python3.12, both are unified to `python3.12`